sql/chunk_constraint.sql

```diff
@@ -29,7 +31,14 @@ BEGIN
         SELECT oid INTO STRICT constraint_oid FROM pg_constraint
         WHERE conname=chunk_constraint_row.hypertable_constraint_name AND
               conrelid = format('%I.%I', hypertable_row.schema_name, hypertable_row.table_name)::regclass::oid;
-        def := pg_get_constraintdef(constraint_oid);
+        SELECT tablespace INTO indx_tablespace FROM pg_indexes
+        WHERE indexname = chunk_constraint_row.hypertable_constraint_name;
```

Comment:
I worry that assuming that the index and constraint names are the same isn't strictly correct and only works incidentally. I think the correct approach is to get the `pg_constraint` row for the constraint and get the index OID (`pg_constraint.coninid`) from there.

Reply:
I believe that constraint name and index name are always the same, but they are stored in different catalog tables. So might be difficult to guarantee.

So implemented the suggestion.